src/t-i18n.ts

```diff
@@ -98,7 +99,7 @@ export class I18n {
 }
 
 function createT(context: I18n): TFunc {
-	let T: TFunc = function translate(message: string, replacements?: (Replacements), id?: string): string {
+	let T: any = function translate(message: string, replacements?: (Replacements), id?: string): string {
```

Comment:
It's too bad this is required. I wonder if we could do `Partial<TFunc>` instead, then `return T as TFunc;` at the end. That would at least give a tad more type safety.

Reply:
I tried Partial TFunc but I think it got confused by the callable function signature. I am not sure how to properly type a function object with properties and then construct it.